pkg/watches/helpers.go

```diff
@@ -171,3 +190,20 @@ func getCscOwnerKey(labels map[string]string) *client.ObjectKey {
 
 	return &client.ObjectKey{Namespace: ownerNamespace, Name: ownerName}
 }
+
+// getOpSrcOwnerKey checks for the OperatorSource owner labels within the labels
+// of the child resource  and computes the key if present. It returns nil if
```

Comment:
nit - there is an extra space between `resource` and `and`.

Reply:
This is also present in the `CatalogSourceConfig` code. I will fix it there too.